__tests__/OmnichannelChatSDK.spec.ts

```diff
@@ -567,6 +567,20 @@ describe('Omnichannel Chat SDK', () => {
             expect(chatSDK.OCClient.getChatConfig.mock.calls[0][1]).toEqual(optionalParams.sendCacheHeaders);
         });
 
+        it('ChatSDK.getChatConfig() failure should throw an exception by default', async () => {
+            const chatSDK = new OmnichannelChatSDK(omnichannelConfig);
+
+            const mockedErrorMessage = "MockedErrorMessage";
+            chatSDK.OCClient = {};
+            chatSDK.OCClient.getChatConfig = jest.fn(() => {throw Error(mockedErrorMessage)});
+
+            try {
+                await chatSDK.getChatConfig();
```

Comment:
Jest has some syntactic sugar for expecting throws
https://jestjs.io/docs/expect#tothrowerror

Reply:
Thanks for sharing! I will add a task to investigate on this since using this new pattern would fail the tests.